ext/nnstreamer/tensor_decoder/tensordec-imagelabel.c

```diff
@@ -202,6 +202,11 @@ il_decode (void **pdata, const GstTensorsConfig * config,
   str = data->labels.labels[max_index];
   size = strlen (str);
 
+  if (!str || size == 0) {
```

Comment:
recommend to change line 203 ~ 205 (IMO, the tool may check strlen (nullptr) case if this condition is added)
```
if (!str || (size = strlen (str)) == 0) {
```

Reply:
looks better